migrations/versions/25e44a14ecb8_added_psi_enabled.py

```diff
@@ -0,0 +1,34 @@
+"""added psi enabled
+
+Revision ID: 25e44a14ecb8
+Revises: 290a91f4eff6
+Create Date: 2022-09-22 15:52:08.936706
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '25e44a14ecb8'
+down_revision = '290a91f4eff6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('performance_profile', 'report_date',
+               existing_type=postgresql.TIMESTAMP(timezone=True),
+               nullable=True)
+    op.add_column('systems', sa.Column('psi_enabled', sa.Boolean(), nullable=True))
+    # ### end Alembic commands ###
```

Comment:
As a thumb rule, both tables `performance_profile` & `performance_profile_history` should be identical in terms of schema. Could you please make change in `performance_profile_history` table as well?

Reply:
I have added the field on Systems, do we need this info. on `performance_profile` and `performance_profile_history`?

Reply:
@kgaikwad Lets check with Pau if we would like to include `psi_enabled` in profile tables as well, thanks!

Reply:
@saltgen I don't really know if we would like to include `psi_enabled` or not. What do we get/lose by adding/not adding it?

Reply:
Adding this as a discussion point for today's Updates call

Reply:
@pgarciaq We have these fields available on every `performance_profile`.

```
performance_record
performance_utilization
report_date
state
operating_system
rule_hit_details
number_of_recommendations
system_id
```
The same record gets copied over to `performance_profile_history` table. 

We do this to keep track of the system changes over time.

W.r.t above do you think `psi_enabled` is something we need to keep track of over the lifetime of a system?

Reply:
As discussed, moving ahead with `psi_enabled` on `performance_profile` and `performance_profile_history` tables, respectively.

Reply:
@kgaikwad Changes pushed